src/tests/end-to-end/common/element-identifiers/fastpass-selectors.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export const fastPassSelectors = {
+    tabStopVisulizationStart: '.insights-tab-stops',
```

Comment:
I think this belongs in `target-page-selectors` - that's where you're looking for it in the test

Reply:
agreed;